scripts/bootstrap/compile.sh

```diff
@@ -17,12 +17,13 @@
 # Script for building bazel from scratch without bazel
 
 if [ -d derived/jars ]; then
-  PROTOBUF_JARS=derived/jars
+  ADDITIONAL_JARS=derived/jars
 else
-  PROTOBUF_JARS="/usr/share/java/protobuf.jar /usr/share/java/protobuf-java-util.jar"
+  DISTRIBUTION=${DISTRIBUTION:-debian}
+  ADDITIONAL_JARS="$(grep -o "\".*\.jar\"" tools/distributions/${DISTRIBUTION}/${DISTRIBUTION}_java.BUILD | sed "s/\"//g" | sed "s/^/\/usr\/share\/java\//g")"
```

Comment:
Please rewrite this like I proposed on the other PR.

Reply:
Done.